.github/workflows/CI.yml

```diff
@@ -360,7 +360,7 @@ jobs:
 
       - name: Build Image
         if: steps.cached_image.outputs.cache-hit != 'true'
-        run: docker build -t $IMAGE_NAME -f cmd/awscollector/Dockerfile .
+        run: docker buildx build --platform linux/amd64 -t $IMAGE_NAME -f cmd/awscollector/Dockerfile .
```

Comment:
Same idea as Jeffrey. Would it be good to pass in after using build-and-push action PR?

Reply:
This line does not need to change I can revert